src/refactor_nrepl/analyzer.clj

```diff
@@ -77,7 +77,8 @@
              (not (util/self-referential? ns)))
     ;; Use `locking`, because AST analysis can perform arbitrary evaluation.
     ;; Parallel analysis is not safe, especially as it can perform `require` calls.
-    (locking core/require-lock
+    (locking core/require-lock ;; for both `require` and `aj/analyze-ns`
+      (require ns) ;; ease the work for t.ana
```

Comment:
Here's a general tip with regards to writing comments: have them answer *why questions*.

Since it's here we can assume that it's needed, but we're still left asking why that is.  You could remedy that with a comment like:  `;; t.ana sometimes fails to require the needed namespaces on its own`

Reply:
I'd use this as the comment here:

```
Performing this require makes it more likely that t.ana will succeed. I believe it's becase require will also require other stuff recursively. t.ana does so in theory as well but it's a bit more rigid.
```